scripts/lib/setup_path.py

```diff
@@ -12,5 +12,8 @@ def setup_path() -> None:
         activate_this = os.path.join(venv, "bin", "activate_this.py")
         activate_locals = dict(__file__=activate_this)
         exec(open(activate_this).read(), activate_locals)
-        if not os.path.exists(activate_locals["site_packages"]):
+        # Check that the python version running this function
+        # is same as python version that created the virtualenv.
+        python_version = "python{0[0]}.{0[1]}".format(sys.version_info[0:2])
```

Comment:
This might as well be one of
* `"python{0[0]}.{0[1]}".format(sys.version_info)`
* `"python{}.{}".format(*sys.version_info[:2])`

Reply:
@andersk Sure I will take care of this from next time.